os/update-strategies.md

```diff
@@ -209,3 +221,5 @@ write_files:
         fi
         exit 0
 ```
+
+[rollback]: /os/docs/latest/manual-rollbacks.html
```

Comment:
These files get rewritten, replacing the .md with .html. The markdown should be valid on it's own.